etc/doc/tutorial/pl/08.5-Ring-Chains.md

```diff
@@ -56,14 +56,16 @@ na jaką nas tylko najdzie ochota. Dzięki temu mamy bardzo bogatą
 i potężną paletę sposobów, dzięki którym możemy tworzyć nowe pierścienie 
 z już istniejących.
 
-## Immutablility
```

Comment:
Ha! Just noticed that this was spelt incorrectly! I guess I'll create a PR for it :-)